ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -598,15 +592,58 @@ private class JdbcLedgerDao(
     )
   }
 
+  /** Prunes the events and command completions tables.
+    *
+    * @param pruneUpToInclusive         Offset up to which to prune archived history inclusively.
+    * @param pruneAllDivulgedContracts  Enables pruning of all immediately and retroactively divulged contracts
+    *                                   up to `pruneUpToInclusive`.
+    *
+    * NOTE:  Pruning of all divulgence events needs to happen taking into account the following considerations:
+    *        1.  Pruning of all divulged contracts needs to use `pruneUpToInclusive`
+    *            beyond `participant_all_divulged_contracts_pruned_up_to_inclusive`
+    *            (i.e. the last successfully used `pruneUpToInclusive` with `pruneAllDivulgedContracts` set).
```

Comment:
Exactly. However, I realized there's a better place for this remark since it is not a constraint on the input arguments but just an info on how pruning will work for disclosures. Will remove from here and add with the disclosure divulgence PR.